docs/animations/AnimationSet.md

```diff
@@ -0,0 +1,121 @@
+| Quartic    | Creates an animation that accelerates or decelerates using the formula f(t) = t4                                                                        | ![QuarticEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/quarticease-graph.png)     |
+| Quintic    | Create an animation that accelerates or decelerates using the formula f(t) = t5                                                                         | ![QuinticEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/quinticease-graph.png)     |
+| Sine       | Creates an animation that accelerates or decelerates using a sine formula                                                                               | ![SineEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/sineease-graph.png)           |
+
+***Note:** EasingType is used only when AnimationSet.UseComposition == false*  
+***Note:** Blur, Light and Saturation animation doesn't support easing*
+
+## Examples
+
+- AnimationSet has endless possibility. Here is an example of creating popup effect
+
+    **Sample Code**
+    ```csharp
+    FrameworkElement preElement = null;
+    private void MyUIElement_PointerEntered(object sender, PointerRoutedEventArgs e)
+    {
+        preElement = sender as FrameworkElement;
+        preElement.Blur(value: 0).Fade(value: 1).Scale(centerX: 100, centerY: 100, easingType: EasingType.Sine);
+                .SetDurationForAll(500);
+                .Start();
+    }
+
+    private void MyUIElement_PointerExited(object sender, PointerRoutedEventArgs e)
+    {
+        if (preElement != null)
+        {
+            preElement.Blur(value: 0).Fade(value: 0.1f).Scale(scaleX: 0.5f, scaleY: 0.5f, centerX: 100, centerY: 100, easingType: EasingType.Sine)
+                    .SetDurationForAll(500);
+                    .Start();
+        }
+    }
+    ```
+    **Sample Output**
+
+    ![Use Case 1 Output](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/AnimationSet/Use-Case-1.gif)
+- Use `Then()` to create a successive animation
+
+    **Sample Code**
+    ```csharp
+    MyUIElement.Blur(value: 10).Fade(value: 0.5f)
+            .Then()
+            .Fade(value: 1).Scale(scaleX: 2, scaleY: 2, centerX: 100, centerY: 100, easingType: EasingType.Sine)
+            .SetDurationForAll(2500)
+            .Start();
+    ```
+    **Sample Output**
+
+    ![Use Case 2 Output](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/AnimationSet/Use-Case-2.gif)
+
+## Requirements
+
+| [Device family](http://go.microsoft.com/fwlink/p/?LinkID=526370) | Universal, 10.0.10586.0 or higher   |
```

Comment:
Can you change the min to 14393, the toolkit's min?

Reply:
Done